lib/async_hooks.js

```diff
@@ -477,7 +477,7 @@ function runCallback(cb, asyncId) {
 module.exports = {
   // Public API
   createHook,
-  currentId,
+  currentAsyncId,
```

Comment:
Because that defaults to non-enumerable, just to avoid a bit of confusion for people inspecting the `require('async_hooks')` object :)